src/test/java/org/itsallcode/openfasttrace/core/TestLocation.java

```diff
@@ -34,7 +34,7 @@
     private static final String PATH = "path";
 
     @Test
-    public void equalsContract()
+    public void testEqualsContract()
```

Comment:
I think `EqualsVerifier` also verifies `hashCode()`. Please check and maybe rename the test.